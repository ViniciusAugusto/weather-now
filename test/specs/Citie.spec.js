import Citie from '@/components/Citie'
import Vuex from 'vuex'
import { shallow, createLocalVue } from 'vue-test-utils'

const localVue = createLocalVue()

localVue.use(Vuex)

describe('Citie.vue', () => {
  let cmp

  beforeEach(() => {
    cmp = shallow(Citie, {
      localVue
    })
  })

  test('colorTemp return class color for temp', () => {
    expect.assertions(3)
    expect(cmp.vm.colorTemp(-1)).toEqual('blue')
    expect(cmp.vm.colorTemp(20)).toEqual('orange')
    expect(cmp.vm.colorTemp(30)).toEqual('red')
  })

  test('has the expected html structure', () => {
    const wrapper = shallow(Citie)
    expect(wrapper.element).toMatchSnapshot()
  })
})
