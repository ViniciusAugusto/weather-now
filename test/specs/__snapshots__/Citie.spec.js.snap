// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Citie.vue has the expected html structure 1`] = `
<div
  class="citie"
/>
`;
